crates/proof-of-sql/src/base/database/owned_and_arrow_conversions.rs

```diff
@@ -1,10 +1,10 @@
 //! This module provide `From` and `TryFrom` implementations to go between arrow and owned types
 //! The mapping is as follows:
-//! OwnedType <-> Array/ArrayRef
-//! OwnedTable <-> RecordBatch
-//! Boolean <-> Boolean
-//! BigInt <-> Int64
-//! VarChar <-> Utf8/String
+//! `OwnedType` <-> `Array/ArrayRef`
+//! `OwnedTable` <-> `RecordBatch`
+//! `Boolean` <-> `Boolean`
+//! `BigInt` <-> `Int64`
+//! `VarChar` <-> `Utf8/String`
 //! Int128 <-> Decimal128(38,0)
 //! Decimal75 <-> S
```

Comment:
Keep things consistent
```suggestion
//! `1nt128` <-> `Decimal128(38,0)`
//! `Decimal75` <-> `S`
```